src/models/converters/transactions/details.rs

```diff
@@ -13,7 +13,17 @@ impl MultisigTransaction {
         info_provider: &mut dyn InfoProvider,
     ) -> Result<TransactionDetails> {
         let safe_info = info_provider.safe_info(&self.safe.to_string())?;
-        let gas_token = self.gas_token.as_ref().map(|token_address| info_provider.token_info(&token_address).ok()).flatten();
+        let gas_token = if let Some(gas_token) = &self.gas_token {
```

Comment:
Wouldn't it make sense to have this logic in the info provider as it applies everywhere?

Reply:
I will push it down then.